tests/storage/blocksd_test.py

```diff
@@ -54,7 +55,7 @@
 from storage.storagefakelib import FakeTaskManager
 
 TESTDIR = os.path.dirname(__file__)
-
+CHUNK_SIZE_MB = blockSD.config.getint("irs", "volume_utilization_chunk_mb")
```

Comment:
Any reason you use blockSD.config instead of importing config and fetching the value from the config?

Reply:
No reason other than to avoid having the extra import. But does no harm and will make it more clear, so I will change it. As it is, the real path to config gets obscured.